### PR TITLE
User prompting from the Command Line to run parsing

### PR DESCRIPTION
**Pull request by ewega**
Added user prompts by way of the command line.
Prompting code can be accessed by `index.js`

**How to Test**

1. Run index.js
2. Input Twitch URL
3. Input Demo Path
4. Input Start Time

_Note 2 and 3 are via text editor to allow copy and pasting. Save the text editor before proceeding (won't actually save somewhere dw)_

### CODE REVIEW

**Review by tjlam — DISMISSED** [review]
I can't seem to run this :( I'm guessing you had to install some node package (inquirer) to get this to work? If so make sure you add it to the `package.json` by doing `npm install --save inquirer`. That way we can run npm install and it will install it for us.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const inquirer = require('inquirer');
```
Comment (tjlam) [inline index.js]: Should this file be in `/demofile_parsing/` I can't get any node stuff to work outside that folder.

Comment (ewega) [reply index.js]: From my end the `index.js` file works outside of demofile-parsing. We can move it if you guys think that's best practice though but it should be working outside _demofile-parsing_

**Comment by ewega** [comment]
@tjlam Made the changes to `Package.json` as you suggested. Good call on that didn't know that was a thing. Not sure if y'all are able to run it now?

**Comment by ewega** [comment]
**Latest Commit:** 

1. Moved `index.js` file to _demofile-parsing folder_ 
2. Added testing boolean to remove redundancy when testing

Sorry for the back and forth PR

File: demofile-parsing/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+const inquirer = require('inquirer');
+
+console.log(`Welcome to the Highlight Generator Tool`);
+console.log();
+var testURL = "https://www.twitch.tv/videos/449439967"; // Change according to Testing needs
+var testFilePath = "C:/Users/eldri/Documents/GitHub/Local_Files/heroic-vs-ence-mirage.dem"; // Change according to Testing needs
+var testStartTime = 2000; // Change according to Testing needs
+
+const TESTING = false; //Turn this on to skip prompts when testing
```
Comment (mattlianje) [inline demofile-parsing/index.js:9]: I like the style of these DEBUG/TESTING bool switches - lets maybe have them ingest a debug arg in the future i.e. `node index.js debug=ON`

**Review by mattlianje — APPROVED** [review]
image

Kinda whack - but as per request of the powerful KKLee and works like a charm. LGTM <3

**Pull request merged**